test/helper.js

```diff
@@ -1,49 +1,106 @@
-let mocha           = require('mocha');
-let debug           = require('debug')('test:helper');
-let load            = require('../src/main');
-let config          = require('typed-env-config');
-let testing         = require('taskcluster-lib-testing');
-let urlencode       = require('urlencode');
-let EventSource = require('eventsource');
-
-const profile = 'test';
-let loadOptions = {profile, process: 'test'};
+const mocha           = require('mocha');
+const debug           = require('debug')('test:helper');
+const load            = require('../src/main');
+const urlencode       = require('urlencode');
+const EventSource     = require('eventsource');
+const libUrls = require('taskcluster-lib-urls');
+const {stickyLoader, Secrets} = require('taskcluster-lib-testing');
 
 // Create and export helper object
-var helper = module.exports = {load, loadOptions};
-
-// Load configuration
-var cfg = config({profile});
+const helper = module.exports;
```

Comment:
If we're going to reference `module.exports` to `helper` then future exports should reference the latter. For example, instead of doing `exports.load` we should do `helper.load`.

Reply:
This was done in [tc-index](https://github.com/taskcluster/taskcluster-index/blob/master/test/helper.js#L12) and other repos too while updating to v10.x

Reply:
Repos like [`tc-queue`](https://github.com/taskcluster/taskcluster-queue/blob/master/test/helper.js) and [`tc-login`](https://github.com/taskcluster/taskcluster-login/blob/master/test/helper.js) reference `helper`. I think we should change `exports` to refer to `helper` unless there is a good reason not to.

Reply:
Dustin is making [changes](https://github.com/taskcluster/taskcluster-queue/pull/279/files#diff-5ab3980e63f908d4556c516d8c88bdc3R22) to tc-queue.

Reply:
Seems like he is doing something similar to you. If this is what the rest of the libraries are looking like then sure, I guess I am okay with it :)

Reply:
Cool :)